trident/line_database.py

```diff
@@ -147,8 +148,8 @@ def __init__(self, input_file=None):
         else:
             self.input_file = 'Manually Entered'
 
-    def add_line(self, element, ion_state, wavelength, gamma,
-                 f_value, field=None, identifier=None):
+    def add_line(self, element, ion_state, wavelength, gamma=None,
```

Comment:
I believe this is leftover in your stack from previous changes you've made on LineTools.  Is it possible to strip this out so we just focus on the Firefly functionality?